packages/netsuite-adapter/src/client/sdf_parser.ts

```
@@ -135,20 +162,31 @@ const transformCustomObject = async (
   )
 }
 
+const safeReadDir = async (dirPath: string): Promise<string[]> => {
+  try {
+    const res = await readDir(dirPath)
+    return res
+  } catch {
+    return []
+  }
+}
+
 export const parseObjectsDir = async (
-  objectsDirPath: string
+  projectPath: string
 ): Promise<CustomTypeInfo[]> => {
-  const filenames = await readDir(objectsDirPath)
+  const objectsDirPath = getObjectsDirPath(projectPath)
+  const filenames = await safeReadDir(objectsDirPath)
```

@omrilit  Should we just check that the Objects dir exists, and if it doesn't return undefined, then in sdf_client throw in that case? Instead of silently return an empty list.